corehq/apps/locations/migrations/0001_initial.py

```diff
@@ -57,7 +56,7 @@ class Migration(migrations.Migration):
                 ('level', models.PositiveIntegerField(editable=False, db_index=True)),
                 ('_products', models.ManyToManyField(to='products.SQLProduct', null=True)),
                 ('location_type', models.ForeignKey(to='locations.LocationType', on_delete=models.CASCADE)),
-                ('parent', mptt.fields.TreeForeignKey(related_name='children', blank=True, to='locations.SQLLocation', null=True, on_delete=models.CASCADE)),
+                ('parent', models.ForeignKey(related_name='children', blank=True, to='locations.SQLLocation', null=True, on_delete=models.CASCADE)),
```

Comment:
how will this migration be run/updated? or is there already a separate migration that handles the conversion (if needed) from `TreeForeignKey` to `ForeignKey`? just want to make sure this will work both on new and existing installs

Reply:
No conversion is necessary because it doesn't do anything special with SQL used in migrations. [`TreeForeignKey`](https://github.com/django-mptt/django-mptt/blob/c72ac22ad04d3bb2c77a6fdbc5f3f17b0dad6ec3/mptt/fields.py#L14-L28) just adds a method for better interaction with django's form widgets.